source/ts/app-connection.ts

```diff
@@ -77,10 +78,6 @@ export class AppConnection extends EventEmitter {
       this.stopServer();
     });
 
-    this.server.on('close', () => {
```

Comment:
Not a particularly useful action. The server shouldn't close without being closed.